packages/snfoundry/contracts/src/SepolliaNFT.cairo

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+// Compatible with OpenZeppelin Contracts for Cairo ^0.13.0
+
+#[starknet::contract]
+mod SepolliaNFT {
+    use openzeppelin::access::ownable::OwnableComponent;
+    use openzeppelin::introspection::src5::SRC5Component;
+    use openzeppelin::token::erc721::ERC721Component;
+    use openzeppelin::token::erc721::ERC721HooksEmptyImpl;
```

Comment:
As specified [here](https://youtu.be/3vn8tgHYKqI?si=41oTI8hnxlCc5gOl&t=44) you need to get rid of this line

Reply:
I thought I deleted that line hehe, it is deleted now! ✅